src/components/pill/pill.spec.js

```diff
@@ -308,6 +571,53 @@ describe('Pill', () => {
           }, fillWrapper);
         });
       });
+
+      describe('when the component size is small', () => {
+        it('matches the expected styles for a small deletable pill', () => {
+          const wrapper = render({
+            children: 'My Text',
+            onDelete: jest.fn(),
+            size: 'small',
+            theme: classicTheme
+          }, TestRenderer.create).toJSON();
+          assertStyleMatch({
+            fontSize: '10px',
+            padding: '2px 16px 2px 8px'
+          }, wrapper);
+        });
+      });
+
+      describe('when the component size is medium', () => {
+        it('matches the expected styles for a medium deletable pill', () => {
+          const wrapper = render({
+            children: 'My Text',
+            onDelete: jest.fn(),
+            size: 'medium',
+            theme: classicTheme
+          }, TestRenderer.create).toJSON();
+          assertStyleMatch({
+            fontSize: '12px',
+            minWidth: '38px',
+            padding: '4px 10px 4px 4px'
+          }, wrapper);
+        });
+      });
+
+      describe('when the component size is large', () => {
```

Comment:
I see that many times there is...
```
description(when the component size is large){
  it('test 1')
}

description(when the component size is large){
  it('test2')
}
```
I would like to ask is there a possbility to do something like

```
description(when the component size is large){
  it('test 1');
  it('test 2');
  it('test 3');
}

```

Reply:
It is good point, unfortunately its separate due to permutations of "isDeletable".